hosts

```diff
@@ -2682,7 +2681,7 @@
 2404:6800:4005:80b::2003 i4.ytimg.com #ytimg.l.google.com
 2404:6800:4005:80b::2003 s.ytimg.com #ytstatic.l.google.com
 
-## Google Video
+﻿## Google Video

```

Comment:
I think this is because Windows saves `01_google.txt` in `UTF-8 with BOM` by default. I resolve it by using sublime